app/controllers/businessmatching/updateservice/AddBusinessTypeHelper.scala

```diff
@@ -135,16 +135,12 @@ class AddBusinessTypeHelper @Inject()(val authConnector: AuthConnector,
     val indices = model.responsiblePeople.fold[Set[Int]](Set.empty)(_.index)
 
     OptionT(dataCacheConnector.update[Seq[ResponsiblePerson]](ResponsiblePerson.key) {
-      case Some(people) if (appConfig.phase2ChangesToggle || isMsbOrTcsp(model)) =>
```

Comment:
the removal of the toggle and isMsbOrTcsp seems to result in the code flow differing from the original code.
and it also throws.
just wondering, is this intentional and are those covered by unit tests before and after the changes?

Reply:
I'll have a look

Reply:
I think that, what's here is right, but I will confirm that

Reply:
I think this code looks ok. 

updateApprovalMsbOrTcsp is trumped by isMsbOrTcsp once the phase 2 flag is removed.